src/component/modal/index.tsx

```diff
@@ -166,6 +171,9 @@ function Modal(props: ModalProps) {
       close={{
         onClick: () => navigate("/"),
       }}
+      close2={{
+        onClick: () => navigate("/"),
+      }}
```

Comment:
`Close2` might be confusing, is this coming from Plasmic? I'd suggest to use another name

Reply:
That's true. I'm not sure why this happened. I'm going to go ahead and merge it to unblock other work, but I will bring it up in the next sync.